README.md

```diff
@@ -21,14 +21,14 @@ will be warned that it is experimental as follows:
 
 Node.js versions 8.12.0 and above provide N-API as a stable feature.
 
-**Branches**
+## **Branches**
```

Comment:
```suggestion
## Branches
```

Then we don't need these asterisks right?

Reply:
Yes I can remove the asterisks from every title. Do you agree?

Reply:
Definitely, yes.